_posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Pulp Container 2.0.0 is Generally Available
```

Comment:
```suggestion
title: Pulp Ansible 0.2.0 is Generally Available
```

Reply:
That's some tasty copy pasta right there. 
I had copied over a header.